utils/proxy/rc_mocked_responses_asm_features.json

```diff
@@ -78,69 +78,70 @@
-		"target_files": [
-			{
-				"path": "datadog/2/ASM_FEATURES/ASM_FEATURES-third/config",
-				"raw": "ewogICJhc20iOiB7CiAgICAgICJlbmFibGVkIjogdHJ1ZQogIH0KfQo="
-			}
-		],
+		"targets": "eyJzaWduZWQiOnsiX3R5cGUiOiJ0YXJnZXRzIiwiY3VzdG9tIjp7Im9wYXF1ZV9iYWNrZW5kX3N0YXRlIjoiZXlKbWIyOGlPaUFpWW1GeUluMD0ifSwiZXhwaXJlcyI6IjIwMzMtMDEtMjNUMTA6MjE6NDMuNTkxNDE5KzAxOjAwIiwic3BlY192ZXJzaW9uIjoiMS4wIiwidGFyZ2V0cyI6eyJkYXRhZG9nLzIvQVNNX0ZFQVRVUkVTL0FTTV9GRUFUVVJFUy1iYXNlL2NvbmZpZyI6eyJjdXN0b20iOnsidiI6Mn0sImhhc2hlcyI6eyJzaGEyNTYiOiJhMzhlYmY5ZmEyNTYwNzFmOTgyM2E1ZjUxMmE4NGU4YTc4NmM4ZGEyYjA3MTk0NTJkZWViNWRjMjg3ZWM5OTBmIn0sImxlbmd0aCI6NDh9LCJkYXRhZG9nLzIvQVNNX0ZFQVRVUkVTL0FTTV9GRUFUVVJFUy1zZWNvbmQvY29uZmlnIjp7ImN1c3RvbSI6eyJ2IjoxfSwiaGFzaGVzIjp7InNoYTI1NiI6IjkyMjFkZmQ5ZjYwODQxNTEzMTNlM2U0OTIwMTIxYWU4NDM2MTRjMzI4ZTQ2MzBlYTM3MWJhNjZlMmYxNWEwYTYifSwibGVuZ3RoIjo0N30sImRhdGFkb2cvMi9BU01fRkVBVFVSRVMvQVNNX0ZFQVRVUkVTLXRoaXJkL3Rlc3RuYW1lIjp7ImN1c3RvbSI6eyJ2IjoxfSwiaGFzaGVzIjp7InNoYTI1NiI6IjFlNGE3NWVkZmQ1ZjY1ZTBhZGVjNzA0Y2M3YzMyZjc5OTg3MTE3ZDg0NzU1NTQ0YWU0OTA1MDQ1Y2RiMGE0NDMifSwibGVuZ3RoIjo0MX0sImRhdGFkb2cvMi9ST0dVRV9QUk9EVUNUL215Y29uZmlnL2NvbmZpZyI6eyJjdXN0b20iOnsidiI6MX0sImhhc2hlcyI6eyJzaGEyNTYiOiI0NTliZDZhNDAzZDhkNTA5YzQ0M2Q3YjQ2NTVkMDI3YWRhM2NhZGUzM2U4YTM5ZjBkNTQ3ZDAyZjRkYjJkM2Q1In0sImxlbmd0aCI6OH0sImRhdGFkb2cvMi9ST0dVRV9QUk9EVUNUL215Y29uZmlnMi9jb25maWciOnsiY3VzdG9tIjp7InYiOjF9LCJoYXNoZXMiOnsic2hhMjU2IjoiYWZlZjkzNDJlNTZmNDQ2NTJhM2E5YWNkZjRiZGZlZGZlOGRhYjJlMDQ5Nzg4NzM0MTRmNjQzOTU0NDRmZjU0MSJ9LCJsZW5ndGgiOjEyfX0sInZlcnNpb24iOjEzfSwic2lnbmF0dXJlcyI6W3sia2V5aWQiOiJlZDc2NzJjOWEyNGFiZGE3ODg3MmVlMzJlZTcxYzdjYjFkNTIzNWU4ZGI0ZWNiZjFjYTI4YjljNTBlYjc1ZDllIiwic2lnIjoiOTRlMWNkZmI3OWVkMDAwYjdhNzFiYmMxMWQ4ZTAyN2FhYTcxOTVkNTBmNDAzNDdlNWI0NTU2YjczY2E0YjQ3MDhiYzg2ODA3ZThmNDI5NDM0MjcwZTRjOWM0YzM1NTAzODZhMWE1ZmRhNzk0MmMyMmNjODAzZWEwNDRmYTVmMDYifV19",
 		"client_configs": [
 			"datadog/2/ASM_FEATURES/ASM_FEATURES-base/config",
 			"datadog/2/ASM_FEATURES/ASM_FEATURES-second/config",
-			"datadog/2/ASM_FEATURES/ASM_FEATURES-third/config"
+			"datadog/2/ASM_FEATURES/ASM_FEATURES-third/testname"
```

Comment:
We need a new client config to validate the new target isn't in target_files.

_12. A new configuration file’s raw bytes are missing from target_files that is referenced by client_configs and targets.signed.targets. This update should fail. The tracer client’s state should not change other than reporting the error in the has_error field along with a message in the error field of the next request."_